src/runner.rs

```diff
@@ -77,7 +77,7 @@ impl Runner {
                 })
                 .collect();
 
-            if !failures.is_empty() {
+            if !failures.is_empty() && mode != &Mode::Overwrite {
```

Comment:
Is there any case where we can still fail with overwrite?

#189 takes care of one.  

A safer approach might be to swallow the errors earlier, where we know what the overwrite did